tests/python/test_templates_prh.py

```
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import re
+from pathlib import Path
+
+
+_PRH_PATH = Path(__file__).resolve().parents[2] / "templates" / "prh.yml"
+
+
+def test_javascript_rule_does_not_use_bare_js_pattern() -> None:
+    content = _PRH_PATH.read_text(encoding="utf-8")
+    # 末尾コメント `- JS # ...` 形式の再導入も検出する．
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `．` (FULLWIDTH FULL STOP). Did you mean `.` (FULL STOP)?

(RUF003)